index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<<head>
```

Comment:
Здесь лишняя угловая скобка
```suggestion
<head>
```